src/esp/physics/bullet/BulletURDFImporter.h

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+// Code adapted from Bullet3/examples/Importers/ImportURDFDemo ...
+
+#ifndef ESP_PHYSICS_BULLET_BULLETURDFIMPORTER_H_
+#define ESP_PHYSICS_BULLET_BULLETURDFIMPORTER_H_
+
+#include <btBulletDynamicsCommon.h>
+#include "BulletDynamics/Featherstone/btMultiBodyDynamicsWorld.h"
+#include "esp/physics/URDFImporter.h"
+namespace esp {
+
+namespace physics {
+
+/**
+ * @brief Structure to hold joint limit constraint info during construction.
+ */
+struct JointLimitConstraintInfo {
+  JointLimitConstraintInfo(int _dof,
+                           float _lowerLimit,
+                           float _upperLimit,
+                           btMultiBodyConstraint* _con) {
```

Comment:
Shouldn't this be set by an initialization list (probably only matters for btMultiBodyConstraint*)